games-emulation/virtualjaguar/virtualjaguar-2.1.2-r2.ebuild

```diff
@@ -26,6 +26,8 @@ DEPEND="${RDEPEND}
 
 S=${WORKDIR}/${PN}
 
+PATCHES=( "${FILESDIR}"/${P}-gcc6.patch )
+
 pkg_pretend() {
 	local ver=4.4
```

Comment:
@Peter-Levine could you remove this pkg_pretend too please? it's from days gone by, and literally everyone has GCC 4.4 at least today.